Sources/StreamVideo/WebRTC/PeerConnection.swift

```diff
@@ -27,8 +27,9 @@ class PeerConnection: NSObject, RTCPeerConnectionDelegate, @unchecked Sendable {
     private var pendingIceCandidates = [RTCIceCandidate]()
     private var publishedTracks = [TrackType]()
     private var screensharingStreams = [RTCMediaStream]()
+    private let badConnectionStates: [RTCIceConnectionState] = [.disconnected, .failed, .closed]
```

Comment:
Can we make this one a set to take advantage of faster lookup operations?

Reply:
it's only 3 elements, doesn't make any difference